keyboards/alpha/layouts/default/28_alpha/keymap.c

```diff
@@ -0,0 +1,82 @@
+#include QMK_KEYBOARD_H
```

Comment:
This should still be located at `keyboards/alpha/keymaps/default/keymap.c`

Reply:
wait i need /keymaps AND /layout? I thought layouts replaced keymaps? Is that all i need in the /keymaps/default and my normal keymap stays in /layouts/28_alpha/default?